xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 11.4.0
+JSONRPC_VERSION 11.4.1
```

Comment:
Needs to be a minor bump e.g. 11.5.0 because this is a non-breaking addition to the API
See https://github.com/xbmc/xbmc/blob/fe157eb163d65461ca4cd94024023df9d72e85d7/xbmc/interfaces/json-rpc/schema/methods.json#L31-L34

Reply:
thx! updated.